python/python/ert/geo/geo_region.py

```diff
@@ -41,10 +45,68 @@ def __init__(self, pointset, preselect=False):
             super(GeoRegion, self).__init__(c_ptr)
         else:
             raise ValueError('Could not construct GeoRegion from pointset %s.' % pointset)
+        self.select_polygon_fn = {
+            (True, True): self._select_inside_polygon,
+            (False, True): self._select_outside_polygon,
+            (True, False): self._deselect_inside_polygon,
+            (False, False): self._deselect_outside_polygon
+        }
+        self.select_halfspace_fn = {
+            (True, True): self._select_above_line,
+            (False, True): self._select_below_line,
+            (True, False): self._deselect_above_line,
+            (False, False): self._deselect_below_line
+        }
+
 
     def getActiveList(self):
         return self._get_index_list()
 
+    def select_polygon(self, polygon, inside=True, select=True):
```

Comment:
Ahhh  - these f...ng names. Initially the ert codebase used `snake_case` - then someone came along from the Java part of the woods and said that `cameCase` was all the rage, and we started a gradual `snake_case` &rightarrow; `camelCase` transition. Since then it seems `camelCase` as fallen out of favor - and we are quite stable at 50% of each.

Personally I have no strong opinion, but:

**I really think we should strive for consistency. The current approach with gradually moving the code in either direction just does not work. We need to agree on or the other and make an all out effort to ensure consistency.**